cli_reference/get_started_cli.adoc

```diff
@@ -52,25 +63,36 @@ users:
 - name: myuser
   user:
     token: NmUyM2M4YjQtZDc2ZC00MDFiLWIxZGEtMjY4N2RlZmJlODE2
----- 
+----
+====
+
+From now on, every subsequent command will make use of the server, session token
+and project referenced in that config file. Notice what happens if you run `osc
+login` again:
 
```

Comment:
This needs to be reworded and brought up to CCS standards. 

[Suggestion]
When you have completed the CLI configuration, every subsequent command uses the configuration file for the server, session token, and project information.

The following example shows the output of the `osc login` command after the CLI has been configured.
[Example]
screen output